store/service.go

```diff
@@ -3,32 +3,35 @@ package store
 import (
 	"context"
 	"errors"
-	"fmt"
 
 	"github.com/heetch/regula"
 	"github.com/heetch/regula/rule"
 )
 
 // Common errors.
 var (
-	ErrNotFound             = errors.New("not found")
-	ErrNotModified          = errors.New("not modified")
+	ErrRulesetNotFound      = errors.New("ruleset not found")
+	ErrRulesetNotModified   = errors.New("not modified")
+	ErrSignatureNotFound    = errors.New("signature not found")
 	ErrInvalidContinueToken = errors.New("invalid continue token")
+	ErrAlreadyExists        = errors.New("already exists")
 )
 
 // RulesetService manages rulesets.
 type RulesetService interface {
-	// Get returns the ruleset related to the given path. By default, it returns the latest one.
-	// It returns the related ruleset version if it's specified.
+	// Create a ruleset entry using a signature.
```

Comment:
I am not fan of this comment. Maybe we should specify that it creates a "bare" ruleset?